tests/regression/roles/sonic_bgp_neighbors/defaults/main.yml

```diff
@@ -80,6 +83,47 @@ deleted_tests:
             v6only: false
 
   - name: test_case_del_02
+    description: Delete BGP peer-group prefix-list attributes
```

Comment:
Instead of only having a test case that deletes all bottom level attributes for both IP AFI/SAFIs, it would be good to have a test case that verifies deletion of groups of attributes. (ip_afi and prefix_limit). Since the instructions for handling of each of the AFI/SAFIs are the same, it would be adequate to only test bottom level attribute deletion for one AFI/SAFI and to do group deletion for the other one.

Reply:
See comments on the related issue of "bulk deletion" of the contents of a dictionary that is not a list item: Because this is not a conveniently supported capability and no support for it is currently provided in other resource modules, I am dropping this test case request.